gui/src/components/settings/pages/GeneralSettings.tsx

```diff
@@ -116,7 +117,12 @@ const defaultValues = {
     mountingResetTaps: 3,
   },
   legTweaks: { correctionStrength: 0.3 },
-  interface: { devmode: false, watchNewDevices: true, feedbackSound: true },
+  interface: {
+    devmode: false,
+    watchNewDevices: true,
+    feedbackSound: true,
+    feedbackSoundVolume: 0.8,
```

Comment:
Honestly it breaks my ears from how loud it was/is by default.
```suggestion
    feedbackSoundVolume: 0.5,
```